packages/office-addin-debugging/src/stop.ts

```diff
@@ -29,7 +31,7 @@ export async function stopDebugging(manifestPath: string, unregisterCommandLine?
             console.log(`Unable to unregister the Office Add-in. ${err}`);
         }
     }
-
+ 
     if (process.env.OfficeAddinDevServerProcessId) {
```

Comment:
You can combine writing and reading the process id whether from env var or file.